docs/file-stores.md

```diff
@@ -0,0 +1,252 @@
+---
+id: file-stores
+title: The Nimbella File Stores
+sidebar_label: Nimbella File Stores
+---
+
+## The Nimbella File Stores
+
+A Nimbella account that has been provisioned with "storage" will have two file stores associated with it.  Often this kind of store is called an "object" store or "blob" store.  It is designed to hold unstructured items of varying size, named and managed as files.
+
+1.  The _web_ store contains files to be served as the front-end portion of your app.
+2.  The _data_ store contains files managed by the actions of your app.
+
+The `nim` command will show the current contents of these stores.
+
+```
+# To show web store contents
+nim web list
+
+# To show data store contents (may be empty initially)
+nim object list
+```
+
+Yes, we use `object` to refer to the data store.  The term `data` would be too generic in this context.
+
+This document divides into these parts.
+
+- [Conceptual Overview](#conceptual-overview)
+- Using a file store from actions via a [Nimbella SDK](#using-file-stores-via-the-nimbella-sdk).
+   - [The `nodejs` SDK](#the-nodejs-sdk)
+   - [The `php` SDK](#the-php-sdk)
+   - [The `python` SDK](#the-python-sdk)
+- Managing a file store with the [`nim` CLI](#managing-file-stores-with-the-nim-CLI).
+
+### Conceptual overview
+
+An file store behaves like a file _system_ in many ways.
+
+- the entities in a file store are often called "files", and they have names, metadata, and contents, just like files in a filesystem
+- the names are hierarchical, separated by slashes and can be used to organize information into a hierarchy
+    - however, unlike in a file system, the "directories" are just portions names: they cannot be empty and are not created or deleted explicitly
+
+The two file stores provided with your Nimbella namespace are similar in some ways and specialized in others.
```

Comment:
Wondering if it's more efficient to communicate the prose with a table.

Reply:
Probably.  I'll look into it.

Reply:
I've simplified the section instead.   I think to communicate the same information in a table might be ungainly.